### ISSUE TITLE
In SILENT mode, reports are sent only on application restart

### ISSUE THREAD

**Issue by GoogleCodeExporter** [useataaaaa/acra#36]
```
With ACRA 3.1.0, using SILENT reporting mode, only explicit silent reports 
(cast with handleSilentException()) are sent immediately. Others are sent on 
application restart.
```

Original issue reported on code.google.com by `kevin.gaudin` on 17 Feb 2011 at 6:55

**Comment by GoogleCodeExporter** [comment]
Original comment by `kevin.gaudin` on 17 Feb 2011 at 6:56
- Added labels: **Milestone-Release3.1.1**
- Removed labels: **Milestone-Release3.2.1**

**Comment by GoogleCodeExporter** [comment]
```
Just released a 3.1.1 version. See Downloads page.
```

Original comment by `kevin.gaudin` on 18 Feb 2011 at 1:40
- Changed state: **Verified**
